### PR TITLE
quickstart updates all round

### PR DESCRIPTION
**Pull request by joshuafernandes**
## PR description
- updating the hdwallet provider package in the 'how to'
- updating the ports of the jsonrpc
- fixed port for explorer

### CODE REVIEW

File: docs/Tutorials/Quickstarts/Private-Network-Quickstart.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ To run this tutorial, you must have the following installed:
 - [Docker and Docker-compose](https://docs.docker.com/compose/install/) 
 
     !!! important 
-        If using MacOS, enable Docker to use up to 4GB of memory on the [_Advanced_ tab in _Preferences_](https://docs.docker.com/docker-for-mac/).  
+        If using MacOS, enable Docker to use up to 4G of memory or 6G of memory if running Privacy examples. The memory settings are found in the [_Advanced_ tab in _Preferences_](https://docs.docker.com/docker-for-mac/).
```
Comment (NicolasMassart) [inline docs/Tutorials/Quickstarts/Private-Network-Quickstart.md]: according to Microsoft Guidelines that we use https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/bits-bytes-terms and https://en.wikipedia.org/wiki/Units_of_information#Systematic_multiples the unit should be `GB` and not only the multiplying prefix `G`.

File: docs/Tutorials/Quickstarts/Private-Network-Quickstart.md [hunk 2]
```diff
@@ -430,7 +430,7 @@ Once complete and successful, the status of the pet you adopted will show 'Succe
 
 ![Dapp UI](../../images/dapp-ui.png)
  
-You can also search for the transaction and view its details in the Block Explorer UI 
+You can also search for the transaction and view its details in the Block Explorer UI at `http://localhost:25000/`
```
Comment (NicolasMassart) [inline docs/Tutorials/Quickstarts/Private-Network-Quickstart.md]: if you don't use code back ticks, the link will be clickable.

File: docs/Tutorials/Quickstarts/Private-Network-Quickstart.md (hunk 1 above)
Comment (joshuafernandes) [reply docs/Tutorials/Quickstarts/Private-Network-Quickstart.md]: I'll update to GB :)

File: docs/Tutorials/Quickstarts/Private-Network-Quickstart.md (hunk 2 above)
Comment (joshuafernandes) [reply docs/Tutorials/Quickstarts/Private-Network-Quickstart.md]: :+1:

**Pull request merged**